diff -Nur a/src/main/java/org/gentoo/java/ebuilder/maven/MavenParser.java format_xml/src/main/java/org/gentoo/java/ebuilder/maven/MavenParser.java
--- a/src/main/java/org/gentoo/java/ebuilder/maven/MavenParser.java	2019-11-27 17:49:27.000000000 +0800
+++ format_xml/src/main/java/org/gentoo/java/ebuilder/maven/MavenParser.java	2020-05-19 19:52:16.245902300 +0800
@@ -102,6 +102,9 @@
                 pomFile.toString(), "help:effective-pom",
                 "-Doutput=" + outputPath);
         processBuilder.directory(config.getWorkdir().toFile());
+        final ProcessBuilder xmlBuilder = new ProcessBuilder("/usr/lib/java-ebuilder/simple_formatter",
+                "" + outputPath);
+        xmlBuilder.directory(config.getWorkdir().toFile());
 
         final Process process;
 
@@ -119,6 +122,20 @@
             Runtime.getRuntime().exit(1);
         }
 
+        final Process xmlProcess;
+        try {
+            xmlProcess = xmlBuilder.start();
+        } catch (final IOException ex) {
+            throw new RuntimeException("Failed to format xml", ex);
+        }
+        try {
+            xmlProcess.waitFor(10, TimeUnit.MINUTES);
+        } catch (final InterruptedException ex) {
+            config.getErrorWriter().println("ERROR: xml formatter run 10 minutes??");
+            Runtime.getRuntime().exit(1);
+        }
+
+
         if (process.exitValue() != 0) {
             config.getErrorWriter().println(
                     "ERROR: Failed to run mvn command:");
