diff -Nur a/src/main/java/org/gentoo/java/ebuilder/maven/MavenCache.java print-groupid/src/main/java/org/gentoo/java/ebuilder/maven/MavenCache.java
--- a/src/main/java/org/gentoo/java/ebuilder/maven/MavenCache.java	2019-11-27 17:49:27.000000000 +0800
+++ print-groupid/src/main/java/org/gentoo/java/ebuilder/maven/MavenCache.java	2020-05-19 16:00:02.443037752 +0800
@@ -40,6 +40,7 @@
      */
     public String getDependency(final String groupId, final String artifactId,
             final String version) {
+
         final Map<String, List<CacheItem>> artifactIds = cache.get(groupId);
 
         if (artifactIds == null) {
@@ -98,6 +99,8 @@
 
         cache.clear();
 
+	System.out.println(config.getCacheFile());
+
         try (final BufferedReader reader = new BufferedReader(
                 new FileReader(config.getCacheFile().toFile()))) {
             String line = reader.readLine();
@@ -134,6 +137,13 @@
             }
         }
 
+	System.out.println("123123123123123123123");
+	System.out.println("123123123123123123123");
+	System.out.println("123123123123123123123");
+	for (Map.Entry<String, Map<String, List<CacheItem>>> entry: cache.entrySet()) {
+		System.out.println(entry.getKey());
+	}
+
         config.getStdoutWriter().println("done");
     }
 
