--- java-pkg-simple.eclass.orig	2020-05-30 14:07:13.402563702 +0800
+++ java-pkg-simple.eclass	2020-05-30 14:09:16.026562572 +0800
@@ -86,6 +86,7 @@
 # classes to ${JAVA_JAR_FILENAME}.
 java-pkg-simple_src_compile() {
 	local sources=sources.lst classes=target/classes apidoc=target/api
+	local getjars_args="--with-dependencies"
 
 	# auto generate classpath
 	java-pkg_gen-cp JAVA_GENTOO_CLASSPATH
@@ -97,7 +98,11 @@
 	# compile
 	local classpath="${JAVA_GENTOO_CLASSPATH_EXTRA}" dependency
 	for dependency in ${JAVA_GENTOO_CLASSPATH}; do
-		classpath="${classpath}:$(java-pkg_getjars ${dependency})" \
+		classpath="${classpath}:$(java-pkg_getjars ${getjars_args} ${dependency})" \
+			|| die "getjars failed for ${dependency}"
+	done
+	for dependency in ${JAVA_CLASSPATH_EXTRA}; do
+		classpath="${classpath}:$(java-pkg_getjars ${getjars_args} ${dependency})" \
 			|| die "getjars failed for ${dependency}"
 	done
 	while [[ $classpath = *::* ]]; do classpath="${classpath//::/:}"; done
